generator/regions/to_string_policy.cpp

```diff
@@ -69,7 +78,11 @@ std::string JsonPolicy::ToString(NodePath const & path) const
   ToJSONObject(*feature, "geometry", geometry);
   ToJSONObject(*feature, "properties", properties);
 
-  auto const cstr = json_dumps(feature.get(), JSON_COMPACT);
+  unsigned int jsonFlags = JSON_COMPACT;
```

Comment:
we don't use `int`, `unsigned int`, `long long` and so on types in our project
use 'int32_t`, `uint64_t` and so instead

Reply:
fixed